### DISCUSSION TITLE
Restrict view to Max Bounds

### DISCUSSION THREAD

**Discussion by Godhran** [rnmapbox/maps#1381] [Q&A]
Is there any way to restrict the view to the max bounds? I've tried doing it on region change but my method of detecting visible bounds, comparing it to maxbounds and using moveTo didn't feel good.

**Comment by ferdicus** [comment]
Maybe I misunderstand, but what's wrong with `Camera` => `maxBounds`?  
Or maybe I'm not getting the use case - sorry

**Reply by Godhran** [reply]
I'm using a raster tile source which is only a small portion of the world. I've tried setting the maxBounds to the most NE and SW points of the tiles but when I zoom in then I lose the ability to move closer to the edge of the map as if there is some padding on the camera. No padding is set on the camera at all.
